cypress-tests/cypress/support/redirectionHandler.js

```diff
@@ -395,6 +395,19 @@ function upiRedirection(
 }
 
 function verifyReturnUrl(redirection_url, expected_url, forward_flow) {
+  const urlParams = new URLSearchParams(redirection_url.search);
+  const paymentStatus = urlParams.get('status');
```

Comment:
i tested your changes and it does not work. the reason being, `redirection_url` is not `hyperswitch.io` at this moment. a wait of at least 10 sec is needed.

Reply:
I have the wait time and necessary changes . @pixincreate  Review it